server/src/main.ts

```diff
@@ -86,15 +73,28 @@ async function bootstrapImmichAdmin() {
   await CommandFactory.run(ImmichAdminModule);
 }
 
+function bootstrapMicroservicesWorker() {
+  const worker = new Worker('./dist/workers/microservices.js');
+  worker.on('exit', (exitCode) => {
+    if (exitCode !== 0) {
+      console.error(`Microservices worker exited with code ${exitCode}`);
+      process.exit(exitCode);
+    }
+  });
+}
+
 function bootstrap() {
   switch (immichApp) {
     case 'immich': {
       process.title = 'immich_server';
+      if (process.env.INTERNAL_MICROSERVICES === 'true') {
```

Comment:
Do we eventually want to have an env var for each worker, or rather something like `IMMICH_ENABLED_WORKERS=api,web,microservices`?

Reply:
We can figure this out later, but a pattern like that might make sense